code/xcvm/lib/core/src/cosmwasm.rs

```diff
@@ -99,6 +99,9 @@ impl KeyDeserialize for InterpreterOrigin {
 	fn from_vec(value: Vec<u8>) -> StdResult<Self::Output> {
 		<(u32, Vec<u8>, Vec<u8>) as KeyDeserialize>::from_vec(value)
 	}
+
+const KEY_ELEMS: u16 = 1234;
```

Comment:
What’s this constant?

Reply:
this is amount of sub keys in key. new features. forgot to set 1. was just dummy war

Reply:
I’m still confused.  This implements [KeyDeserialize](https://docs.rs/cw-storage-plus/latest/cw_storage_plus/trait.KeyDeserialize.html) but I don’t see it defining that constant.